src/main/java/htsjdk/samtools/SAMTagUtil.java

```diff
@@ -33,70 +33,113 @@
+    @Deprecated
+    public static final short GS = SAMTag.GS.getBinaryTag();
+    /** @deprecated reserved tag for backwards compatibility only */
+    @Deprecated
+    public static final short GQ = SAMTag.GQ.getBinaryTag();
+    public static final short LB = SAMTag.LB.getBinaryTag();
+    public static final short H0 = SAMTag.H0.getBinaryTag();
+    public static final short H1 = SAMTag.H1.getBinaryTag();
+    public static final short H2 = SAMTag.H2.getBinaryTag();
+    public static final short HI = SAMTag.HI.getBinaryTag();
+    public static final short IH = SAMTag.IH.getBinaryTag();
+    public static final short MC = SAMTag.MC.getBinaryTag();
+    /** @deprecated reserved tag for backwards compatibility only */
+    @Deprecated
+    public static final short MF = SAMTag.MF.getBinaryTag();
+    public static final short MI = SAMTag.MI.getBinaryTag();
+    public static final short MD = SAMTag.MD.getBinaryTag();
+    public static final short MQ = SAMTag.MQ.getBinaryTag();
+    public static final short NH = SAMTag.NH.getBinaryTag();
+    public static final short NM = SAMTag.NM.getBinaryTag();
+    public static final short OQ = SAMTag.OQ.getBinaryTag();
+    public static final short OP = SAMTag.OP.getBinaryTag();
+    public static final short OC = SAMTag.OC.getBinaryTag();
+    public static final short OF = SAMTag.OF.getBinaryTag();
+    public static final short OR = SAMTag.OR.getBinaryTag();
+    public static final short OX = SAMTag.OX.getBinaryTag();
+    public static final short PG = SAMTag.PG.getBinaryTag();
+    public static final short PQ = SAMTag.PQ.getBinaryTag();
+    public static final short PT = SAMTag.PT.getBinaryTag();
+    public static final short PU = SAMTag.PU.getBinaryTag();
+    public static final short QT = SAMTag.QT.getBinaryTag();
+    public static final short Q2 = SAMTag.Q2.getBinaryTag();
+    public static final short QX = SAMTag.QX.getBinaryTag();
+    public static final short R2 = SAMTag.R2.getBinaryTag();
+    public static final short RG = SAMTag.RG.getBinaryTag();
+    /** @deprecated use BC instead */
+    @Deprecated
+    public static final short RT = SAMTag.RT.getBinaryTag();
+    public static final short RX = SAMTag.RX.getBinaryTag();
+    /** @deprecated reserved tag for backwards compatibility only */
+    @Deprecated
+    public static final short S2 = SAMTag.S2.getBinaryTag();
+    public static final short SA = SAMTag.SA.getBinaryTag();
+    public static final short SM = SAMTag.SM.getBinaryTag();
+    /** @deprecated reserved tag for backwards compatibility only */
+    @Deprecated
+    public static final short SQ = SAMTag.SQ.getBinaryTag();
+    public static final short TC = SAMTag.TC.getBinaryTag();
+    public static final short U2 = SAMTag.U2.getBinaryTag();
+    public static final short UQ = SAMTag.UQ.getBinaryTag();
 
-    private static SAMTagUtil singleton;
+    private static final SAMTagUtil singleton = new SAMTagUtil();
```

Comment:
Conventionally static fields are all caps
```suggestion
    private static final SAMTagUtil SINGLETON = new SAMTagUtil();
```